doc/examples/example4.md

```diff
@@ -68,12 +68,42 @@ uRZLQUBt1w+r1qEakvSIoinjrmS616qfkOBPHJEkvQ==
 -----END CERTIFICATE-----)"
 };
 \endcode
-
+In the example bellow we are loading two certificates from their respective PEM strings
+and then are verifying if the intCert was signed by the rootCert
 \code{.cpp}
-auto rootCert = X509Certificate::fromPEM("rootPemString");
-auto intCert = X509Certificate::fromPEM("intPemString");
+X509Certificate rootCert = X509Certificate::fromPEM("rootPemString");
+X509Certificate intCert = X509Certificate::fromPEM("intPemString");
 
 void X509Certificate::verify(rootCert, intCert);
 \endcode
- 
+
+
+
+X509Certificate class also provides methods to get various types of information like issuer
+and subject distinguished name, certificate validity, serial number and public key.
+\code{.cpp}
+X509Certificate cert = X509Certificate::fromPEM(intPemString);
+
+DistinguishedName issDN = cert.getIssuerDistinguishedName();
+DistinguishedName subjDN = cert.getSubjectDistinguishedName();
+uint64_t serialN = cert.getSerialNumber();
+std::chrono::system_clock::time_point validity = X509Certificate::getNotAfter();
+AsymmetricPublicKey pubK = cert.getPublicKey();
+\endcode
+
+A certificate can be written to a PEM format. 
+\code{.cpp}
+X509Certificate cert; 
+std::string certToPem = cert.toPEM();
+\endcode
+
+It's possible to add more parameters to the Certificate verification by providing the verify method
+with a VerificationCtx Object where we can set a CRL (certificate revocation list),
+a time check verification and others.
```

Comment:
Add this line here: "In the example bellow we can see an example where we create a context that enables a time check verification."

Reply:
Done.